README.md

```diff
@@ -20,7 +20,7 @@ You can use React as a `<script>` tag from a [CDN](https://reactjs.org/docs/cdn-
 
 ## Documentation
 
-You can find the React documentation [on the website](https://reactjs.org/).  
+You can find the latest React documentation [on the React website](https://reactjs.org/docs).
```

Comment:
```suggestion
You can find the latest React documentation [on the React website](https://react.dev/).
```

Reply:
Thank you for correcting.. You're absolutely right